test.js

```diff
@@ -1,9 +1,14 @@
 var tape = require('tape')
+var Nanobus = require('nanobus')
+require('jsdom-global')(null, { url: 'http://localhost/' })
+window.localStorage = window.localStorage || {} // See: yoshuawuyts/nanotiming#7
+window.requestAnimationFrame = window.requestAnimationFrame || function (cb) { process.nextTick(cb) }
```

Comment:
Oh heck yeah, that'd make it easier. At least 1 other test simply wouldn't work otherwise too. Glad to hear you're good using that.